### PR TITLE
Add missing feature designators (#657).

### LINKED ISSUE #657: Define missing feature designators.
Ensure that all new features are covered by at least one feature designator, adding as needed to the following task list:

- [x] ``#condition-media`` - media function used with ``condition``
- [ ] ``#display-inlineBlock`` - use of ``tts:display='inlineBlock'``
- [ ] ``#fontFamily-p`` - ``tts:fontFamily`` applies to ``p``
- [ ] ``#fontSize-p`` - ``tts:fontSize`` applies to ``p``
- [ ] ``#fontStyle-p`` - ``tts:fontStyle`` applies to ``p``
- [ ] ``#fontWeight-p`` - ``tts:fontWeight`` applies to ``p``
- [x] ``#set-multiple-styles`` - use of multiple style properties on ``set``

A checked box above means that the feature has been added into a PR associated with this issue.

Note that the level of granularity for defining feature designators is not fixed. Some _features_ may be defined at a finer degree of granularity than others. For example, in general, defining a feature designator for every possible value of a syntactic feature is not recommended.

### PR DESCRIPTION
**Pull request by skynavga**
Closes #657.

### CODE REVIEW

**Comment by skynavga** [comment]
I am not proposing merging this until all the missing feature designators are added.

**Comment by skynavga** [comment]
@nigelmegitt added the remaining designators

File: spec/ttml2.xml [hunk 1]
```diff
@@ -19309,6 +19309,10 @@ attribute.</p>
 for all defined values of the <loc
 href="#style-attribute-display"><att>tts:display</att></loc>
 attribute.</p>
+<p>Notwithstanding the above, support for the <code>#display</code> feature does not require support for the following feature:</p>
+<ulist>
+<item><p><loc href="#feature-fontFamily-p"><code>#display-inlineBlock</code></loc></p></item>
```
Comment (cconcolato) [inline spec/ttml2.xml]: wrong link

File: spec/ttml2.xml [hunk 2]
```diff
@@ -19339,6 +19343,24 @@ support for all defined values of the <loc
 href="#style-attribute-display"><att>tts:display</att></loc>
 attribute when applied to a <loc href="#terms-content-element">content element</loc> that generates an
 inline area.</p>
+<p>Notwithstanding the above, support for the <code>#display-inline</code> feature does not require support for the following feature:</p>
+<ulist>
+<item><p><loc href="#feature-fontFamily-p"><code>#display-inlineBlock</code></loc></p></item>
```
Comment (cconcolato) [inline spec/ttml2.xml]: wrong link

File: spec/ttml2.xml (hunk 1 above)
Comment (skynavga) [reply spec/ttml2.xml]: fixed

File: spec/ttml2.xml (hunk 2 above)
Comment (skynavga) [reply spec/ttml2.xml]: fixed

**Comment by skynavga** [comment]
Addressed all comments, and merging (early).

**Comment by skynavga** [comment]
See https://lists.w3.org/Archives/Public/public-tt/2018May/0103.html regarding possible reason not to define `fontFamily-p`, `fontSize-p`, `fontStyle-p`, and `fontWeight-p`.

**Pull request merged**